developing/languages/BASH.md

```diff
@@ -45,3 +57,5 @@ Shell programming is deceptively perilous (see [Bash Pitfalls](https://mywiki.wo
 * [Bash FAQ](http://mywiki.wooledge.org/BashFAQ)
 * [Bash Pitfalls](https://mywiki.wooledge.org/BashPitfalls)
 * [Bash-Hackers Wiki](http://wiki.bash-hackers.org/)
+* [Path Variables](https://truss.works/blog/2016/2/26/engineer-how-to-access-and-edit-your-path-system-variable)
+* [CLI Navigation Shortcuts](https://www.gnu.org/software/bash/manual/bash.html#Introduction-and-Notation)
```

Comment:
This seems fine for now, and perhaps later we might add some caveats depending on your OS and CLI app. For example, the bash manual says you can move the cursor between words by using the meta key + f or b, but that doesn't work on a Mac with Terminal or iTerm out of the box. In Terminal and iTerm, the default is to use option + left or right arrow. If you want to use option + f or b, there is a checkbox in Terminal app to "Use option as Meta key" under Preferences -> Profiles -> Keyboard.

Reply:
Got it. Might want to make a story for that. This story was just about adding these two resources..